code/modules/mob/living/say.dm

```diff
@@ -347,7 +347,10 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 
 /mob/living/proc/get_message_language(message)
 	if(message[1] == ",")
-		var/key = message[1 + length(message[1])]
+		//skyrat edit - fucking hate this i want to use multichar keys
+		var/list/bruh = splittext(message, " ", 2)
```

Comment:
This doesn't feel too sane, could imagine this being null etc

Reply:
ok then what do i do

Reply:
ok wait hold on i'll just add a check for length